### PR TITLE
Add the context parameter to the mfd configuration hooks

### PR DESCRIPTION
**Pull request by williamjamir**
ASIM-4008

### CODE REVIEW

File: HISTORY.rst [hunk 1]
```diff
@@ -2,6 +2,12 @@
 History
 =======
 
+0.7.1 (Unreleased)
+==================
+
+* Add support for context on alfasim_configure_fields, alfasim_configure_layers and alfasim_configure_phases
```
Comment (nicoddemus) [inline HISTORY.rst:8]:
```suggestion
* Add context support on ``alfasim_configure_fields``, ``alfasim_configure_layers`` and ``alfasim_configure_phases``.
```

File: src/_alfasim_sdk/version.py [hunk 2]
```diff
@@ -1 +1 @@
-__version__ = "0.7.0"
+__version__ = "0.7.1"
```
Comment (nicoddemus) [inline src/_alfasim_sdk/version.py:1]:
```suggestion
__version__ = "0.8.0"
```

Being able to receive a context is definitely a new feature. 👍

**Review by nicoddemus — CHANGES_REQUESTED** [review]

**Review by nicoddemus — APPROVED** [review]

File: src/_alfasim_sdk/hook_specs_gui.py [hunk 3]
```diff
@@ -248,13 +248,17 @@ def alfasim_configure_fields():
      - Phase, defined by :class:`~alfasim_sdk.types.AddPhase` or :class:`~alfasim_sdk.types.UpdatePhase`.
      - Layer, defined by :class:`~alfasim_sdk.types.AddLayer` or :class:`~alfasim_sdk.types.UpdateLayer`.
 
+     The ``ctx`` parameter is provided to retrieve information about the current state of the application and the current value
```
Comment (oliveira-mauricio) [inline src/_alfasim_sdk/hook_specs_gui.py:251]: There is an extra space 🤷🏼‍♂️

**Review by oliveira-mauricio — APPROVED** [review]
The plugins should be updated too.

**Review by prusse-martin — APPROVED** [review]

**Pull request merged**